x/datadeal/keeper/deal.go

```diff
@@ -14,6 +15,11 @@ import (
 	oracletypes "github.com/medibloc/panacea-core/v2/x/oracle/types"
 )
 
+const (
+	BlockPeriod           = 6 * time.Second
+	DealDeactivationParam = int64(3) // Todo: make DealDeactivationParam to Params of DataDeal
```

Comment:
I think use much appropriate variable name such as `DeactivationPeriodMultiplier`.
If you are gonna move this to module params, it doesn't matter for now.

Reply:
The naming you suggested looks better. If it's not important, I think it's okay to change it all at once in refactoring later.